src/DI/WebpackExtension.php

```diff
@@ -32,6 +32,7 @@ class WebpackExtension extends CompilerExtension
 		'devServer' => [
 			'enabled' => NULL,
 			'url' => NULL,
+			'proxy' => NULL,
```

Comment:
What do you think about renaming it to `publicUrl`? I mean, it needs an explanation in the README anyway, but "proxy" seems too vague to me

Reply:
https://github.com/o2ps/WebpackNetteAdapter/pull/13/commits/9e15518c6373f5a3bcf40b558fc54ee527ef3847